### PR TITLE
ELK and DB doc contribution

### PR DESCRIPTION
**Pull request by mosabua**
Replaces https://github.com/oneops/oneops.github.io/pull/6

Once I am done where I will assign to @leelakumili for final input regarding my questions. I did all the updates from the old PR.

### CODE REVIEW

File: user/howto/add-elk-stack-to-application.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The following steps are an example on how to configure Logstash to collect the Tomcat access log.
+
+1. Add a Logstash component under the Tomcat platform. 
+
+1. Edit the inputs, filters and outputs option as required.
+
+1. Deploy the Logstash component.
+
+1. Verify Logstash started successfully without errors.
```
Comment (mosabua) [inline user/howto/add-elk-stack-to-application.md]: How @leelakumili ?

File: user/howto/add-elk-stack-to-an-application.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This page details how to add [Elasticsearch](https://www.elastic.co/products/elasticsearch),
+[Logstash](https://www.elastic.co/products/logstash) and [Kibana](https://www.elastic.co/products/kibana)
+- the Elastic stack or ELK stack - to an application.
+
+# Elasticsearch Setup
+
+1. Add a new platform for Elasticsearch using the 'Elasticsearch with LB' pack.
+
+1. If required, edit the configuration under the Elasticsearch component.
```
Comment (mosabua) [inline user/howto/add-elk-stack-to-an-application.md]: Example?

File: user/howto/add-elk-stack-to-an-application.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+1. Verify Kibana by accessing the user interface at http://ipaddress:5601/app/kibana
+
+
+# Logstash Setup
+
+The following steps are an example on how to configure Logstash to collect the Tomcat access log.
+
+1. Add a Logstash component under the Tomcat platform.
+
+1. Edit the inputs, filters and outputs options as required.
```
Comment (mosabua) [inline user/howto/add-elk-stack-to-an-application.md]: Example?

File: user/howto/add-elk-stack-to-an-application.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Deploy the Logstash component.
+
+1. Verify Logstash started successfully without errors.
```
Comment (mosabua) [inline user/howto/add-elk-stack-to-an-application.md]: How?

File: user/howto/add-elk-stack-to-an-application.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+# Validation
+
+1. After the Logstash deployment, verify that indices are created on Elasticsearch at http://ipaddress:9200/_cat/indices
+and that the status is green.
+
+1. Now that logs are parsed and stored in Elasticsearch, you can configure Kibana to generate reports as required.
```
Comment (mosabua) [inline user/howto/add-elk-stack-to-an-application.md]: Example? How? Using the Kibana user interface?

File: user/howto/deploy-application-with-database.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+The purpose of this document is to briefly explain how to configure and deploy a simple web application with a MySql 
+database.
+
+# Highlevel Overview
+
+1.Setup MySQL database on OneOps.
+
+1. Create a schema on MySQL.
+
+1. Configure the web application.
```
Comment (mosabua) [inline user/howto/deploy-application-with-database.md:17]: How? Which one?

**Comment by mosabua** [comment]
@leelakumili  can you please have a look at the current state of the document in this PR and provide some answers to my questions?

File: user/howto/add-elk-stack-to-an-application.md (hunk 2 above)
Comment (lkumili-nisum-com) [reply user/howto/add-elk-stack-to-an-application.md]: Like Change values for Number of shards or Number of replicas or change default directory location for installation,Elasticsearch logs, etc..

File: user/howto/add-elk-stack-to-an-application.md (hunk 3 above)
Comment (lkumili-nisum-com) [reply user/howto/add-elk-stack-to-an-application.md]: `Inputs : file {path => "/opt/tomcat7/logs/access*.log"  sincedb_path => "/opt/logstash/sincedb-access"  }`

`Filters:  grok {     match => {       "message" => "%{IP:client_ip} %{USER:ident} %{USER:auth} \[%{HTTPDATE:apache_timestamp}\] \"%{WORD:method} /%{NOTSPACE:request_page} HTTP/%{NUMBER:http_version}\" %{NUMBER:server_response}"}}`

`Outputs:  if "_grokparsefailure" not in [tags] {     stdout {       codec => rubydebug     }     elasticsearch { host => ["ipaddress"]  protocol => http  port => "port"  }   } `

File: user/howto/add-elk-stack-to-an-application.md (hunk 4 above)
Comment (lkumili-nisum-com) [reply user/howto/add-elk-stack-to-an-application.md]: SSH Tomcat vm where Logstash is installed and tail logstash.log to check no error in log file.

tail -f /var/logstash/logstash.log

File: user/howto/add-elk-stack-to-an-application.md (hunk 5 above)
Comment (lkumili-nisum-com) [reply user/howto/add-elk-stack-to-an-application.md]: For more details refer to: https://www.elastic.co/guide/en/kibana/current/createvis.html

**Comment by lkumili-nisum-com** [comment]
@mosabua I have added the details requested above. Let me know if anymore information required.

**Comment by mosabua** [comment]
Great.. Thanks @lkumili will review, update, merge and publish later today.

File: user/howto/add-elk-stack-to-an-application.md (hunk 2 above)
Comment (mosabua) [reply user/howto/add-elk-stack-to-an-application.md]: Added an example.

File: user/howto/add-elk-stack-to-an-application.md (hunk 3 above)
Comment (mosabua) [reply user/howto/add-elk-stack-to-an-application.md]: Added inputs to keep it simple

File: user/howto/add-elk-stack-to-an-application.md (hunk 4 above)
Comment (mosabua) [reply user/howto/add-elk-stack-to-an-application.md]: Updated.

File: user/howto/add-elk-stack-to-an-application.md (hunk 5 above)
Comment (mosabua) [reply user/howto/add-elk-stack-to-an-application.md]: Added.

**Comment by mosabua** [comment]
Merged and published.

**Pull request merged**